exotica_core/src/task_map.cpp

```diff
@@ -73,6 +73,11 @@ void TaskMap::Update(Eigen::VectorXdRefConst x, Eigen::VectorXdRef Phi, Eigen::M
     if (jacobian.rows() != TaskSpaceDim() && jacobian.cols() != x.rows())
         ThrowNamed("Jacobian dimension mismatch!");
 
+    if (scene_ == nullptr)
```

Comment:
Is this for the case where you create a task map manually without a problem? Or a problem without a proper scene? I am trying to understand how this could be triggered.

Reply:
It's triggered if you overwrite `AssignScene` without properly implementing it. It shouldn't happen that often but it helps to debug the problem.